pkg/policy/mapstate.go

```diff
@@ -147,6 +152,29 @@ func (e *MapStateEntry) RemoveDependent(key Key) {
 	}
 }
 
+func (e *MapStateEntry) getNets(identities Identities, id identity.NumericIdentity) []*net.IPNet {
```

Comment:
The receiver `e` is not used, so maybe this should be a bare function?

Reply:
This is definitely being used now.